fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/TestIsolationExecutionListener.java

```diff
@@ -32,7 +38,24 @@
     @Override
     public void beforeTestMethod(final TestContext testContext) throws Exception {
         final var ocflRepo = getBean(testContext, MutableOcflRepository.class);
-        ocflRepo.listObjectIds().forEach(ocflRepo::purgeObject);
+        final var ocflConfig = getBean(testContext, OcflPropsConfig.class);
+
+        final var hasError = new AtomicBoolean(false);
+
+        ocflRepo.listObjectIds().forEach(object -> {
+            try {
+                ocflRepo.purgeObject(object);
+            } catch (RuntimeException e) {
+                // Recursive deletes don't behave well on Windows and it's possible for the above to error out.
```

Comment:
Did you forget to toggle `hasError` when catching an exception?

Reply:
whoops